content/technology/bpmn/_index.md

```diff
@@ -0,0 +1,55 @@
+---
+title: BPMN 2.0
+description: Business Process Model and Notation (BMN) - Workflow
+weight: 100
+tags: ["bpmn", "workflow", "arbeidsflyt"]
+---
+
```

Comment:
propbably some documentation on the developer handbook too that would help other developers?

Reply:
What kind of documentation where you thinking about?

Reply:
Like where to find the workflow file, how to change/add/delete a step in workflow

Reply:
Will make a new pr for that